views/components/ChartList.jsx

```diff
@@ -45,54 +49,79 @@ const ChartList = () => {
       console.error(error, '*****')
     }
   }
-
+  const handleShareChart = (chart) => {
+    setChartState(chart)
+    setShareChart(true)
+  }
```

Comment:
Do we need both of these state values? Can be boolean value be derived from having a chart set or not?

Reply:
Yes it can be. 
https://github.com/AMP-SCZ/dpdash/pull/131/commits/789e65512c6e640d9852ea9740fdafc3706c462a